src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -177,6 +180,26 @@ const SkillsQuizStepper = () => {
     }
   }, [skillsVisible, selectedSkills]);
 
+  useEffect(() => {
+    const fetchLearnerCourseEnrollments = async () => {
+      try {
+        const response = await fetchCourseEnrollments();
+        const enrolledCourses = camelCaseObject(response.data);
+        const enrolledCourseIds = enrolledCourses.map((course) => course.courseDetails.courseId);
+        skillsDispatch({
+          type: SET_KEY_VALUE,
+          key: 'enrolledCourseIds',
+          value: enrolledCourseIds,
+        });
+      } catch (error) {
+        logError(error);
+      }
+    };
+
+    fetchLearnerCourseEnrollments();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Don't we need to add dependencies?

Reply:
No, as we need to render it once when page loads.